src/calibre/devices/kobo/driver.py

```diff
@@ -1376,6 +1376,7 @@ class KOBOTOUCH(KOBO):
     min_forma_fwversion             = (4, 11, 11879)
     min_librah20_fwversion          = (4, 16, 13337)  # "Reviewers" release.
     min_fwversion_epub_location     = (4, 17, 13651)  # ePub reading location without full contentid.
+    min_fwversion_dropbox           = (4, 17, 13737)  # The Forma only at this point.
```

Comment:
```(4, 18, 13737)```? ;).

Reply:
@davidfor is it 4, 18 or 4, 17?

Reply:
The same as `max_supported_fwversion` (i.e., 4.18).

Reply:
yeah questionis which is correct? max_supported_firmware or this?

Reply:
Well I guess it must be the 18 according to bug https://bugs.launchpad.net/bugs/1846942

Reply:
Yeah, 4.18.